File: bin/travis-encrypt.py
#!/usr/bin/env python

if __name__ == "__main__":

    from Crypto.PublicKey import RSA
    from Crypto.Cipher import PKCS1_OAEP 
    import sys
    import base64

    pub_key = open("etc/travis.key.txt", "r").read()

    imp_key = RSA.importKey(pub_key)

    cipher = PKCS1_OAEP.new(imp_key)
    ciphertext = cipher.encrypt(sys.stdin.read())

    print base64.b64encode(ciphertext)

File: bin/travis-encrypt.rb
#!/usr/bin/env ruby 

require "openssl"
require "base64"

    pub_key = File.open("etc/travis.key.txt", "r").read

    imp_key = OpenSSL::PKey::RSA.new(pub_key)

    puts Base64.encode64(imp_key.public_encrypt(ARGF.read))
